services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/migration/MigrationManagerImplAutowireTest.java

```diff
@@ -209,9 +215,61 @@ public void testGetCount(){
 	public void testGetMaxId() {
 		long mx = migrationManager.getMaxId(adminUser, MigrationType.FILE_HANDLE);
 		assertEquals(Long.parseLong(preview.getId()), mx);
+		assertEquals(startId+1, mx);
 	}
 	
 	@Test

```

Comment:
All these should be moved to unit (non-autowired) test,
mainly to check that security is checked and that
calls to lower layers are made.